rdt/transformers/pii/anonymizer.py

```diff
@@ -98,10 +106,18 @@ def _check_locales(self):
             )
 
     def __init__(self, provider_name=None, function_name=None, function_kwargs=None,
-                 locales=None, enforce_uniqueness=False, missing_value_generation='random'):
+                 locales=None, cardinality_rule=None, enforce_uniqueness=False,
+                 missing_value_generation='random'):
         super().__init__()
+        self._data_cardinality = None
         self.data_length = None
-        self.enforce_uniqueness = enforce_uniqueness
+        self.cardinality_rule = cardinality_rule
```

Comment:
You can do this (in case the user capitalizes the string) 
```suggestion
        self.cardinality_rule = cardinality_rule.lower()
```